File: app/jobs/TickJob.java
package jobs;

import models.Tick;
import play.api.DefaultApplication;
import play.api.Play;
import play.api.Mode;
import play.api.Application;

import java.io.File;


public class TickJob {
    
    public static void main(String[] args) {

        Application application = new DefaultApplication(new File(args[0]), TickJob.class.getClassLoader(), null, Mode.Prod());

        Play.start(application);

        Tick tick = new Tick("Hello from the TickJob");
        tick.save();
    }

}

File: app/jobs/TickJob.scala
package jobs

import modules.TickSchedulerModule
import play.api.inject.guice.GuiceApplicationBuilder
import utils.Tick

object TickJob extends App {

  val app = new GuiceApplicationBuilder(disabled = Seq(classOf[TickSchedulerModule])).build()

  val tick = app.injector.instanceOf[Tick]

  tick.tock()

  app.stop()

}
